src/main/java/eu/dissco/exportjob/domain/JobRequest.java

```diff
@@ -6,7 +6,8 @@
 public record JobRequest(
     List<SearchParam> searchParams,
     TargetType targetType,
-    UUID jobId
+    UUID jobId,
+    Boolean isSourceSystemJob
```

Comment:
Ah, I see why ruff wants to add that trailing comma :laughing: 

In the future, we'll probably want a generic `jobType` enum instead of making this a boolean.

Reply:
I think you can leave it like this if you want or change it now

Reply:
Discussed this and we leave it like this